python/paddle/nn/functional/norm.py

```diff
@@ -116,8 +116,8 @@ def batch_norm(
     x,
     running_mean,
     running_var,
-    weight,
-    bias,
+    weight=None,
+    bias=None,
```

Comment:
the following document description also needs to be modified, eg: `weight(Tensor): The weight tensor of batch_norm, can not be None. bias(Tensor): The bias tensor of batch_norm can not be None.`

Reply:
Meanwhile, because these two parameters(`weight`, `bias`) in the OP definition have not default values, they are either passed to the OP after initialization in the Python layer or modified the OP definition

Reply:
if we allow these two parameters have default values, then should add some test case to make sure that behavior of API meets expectations